Source/WebKit/NetworkProcess/cache/NetworkCacheSpeculativeLoadManager.cpp

```diff
@@ -379,7 +379,10 @@ void SpeculativeLoadManager::registerLoad(GlobalFrameID frameID, const ResourceR
         ASSERT(!m_pendingFrameLoads.contains(frameID));
 
         // Start tracking loads in this frame.
-        auto pendingFrameLoad = PendingFrameLoad::create(protectedStorage(), resourceKey, [this, frameID] {
+        auto pendingFrameLoad = PendingFrameLoad::create(protectedStorage(), resourceKey, [this, weakThis = WeakPtr { *this }, frameID] {
```

Comment:
I doubt this silences the warnings if you capture `this` and `weakThis` here. I suspect you'll have to stop capturing `this` and convert the `weakThis` to a `checkedThis` inside the lambda.